pkg/binutils/release.go

```diff
@@ -96,5 +96,5 @@ func InstallBinary(
 
 	binDir, err := installBinaryWithVersion(app, version, installDir, binPrefix, downloader, installer)
 
-	return version, binDir, err
+	return version, filepath.Join(binDir, "avalanchego"), err
```

Comment:
this func in used to install evm, so it should return binDir not avalanchego path

Reply:
I was talking about SetupAvalanchego func change not InstallBinary func